typeid/typeid-js/src/typeid.ts

```diff
@@ -70,7 +70,7 @@ export class TypeID<const T extends string> {
     return uuid.toString();
   }
 
-  public toString(): string {
+  public toString(): `${T}_${string}` | string {
```

Comment:
Makes sense 👍 , and without testing it I assume typescript can figure out which one is which when resolving?